conf/references/mlss_conf.xml

```diff
@@ -57,94 +62,194 @@
       <genome name="magnaporthe_oryzae"/>
       <genome name="zymoseptoria_tritici"/>
       <genome name="erysiphe_necator_gca_000798715"/>
+      <genome name="schizosaccharomyces_pombe"/>
     </collection>
 
     <collection name="vertebrata">
-      <base_collection name="shared"/>
+      <base_collection name="shared_vertebrates"/>
+      <!-- outgroup -->
+      <genome name="ciona_intestinalis"/>
```

Comment:
```suggestion
```
(Based on including it in the `shared_vertebrates` collection)

Reply:
cannot be removed because not in shared vertebrates